pkg/haprovider/haprovider.go

```diff
@@ -138,16 +138,21 @@ func (r *HAProvider) annotateService(ctx context.Context, cluster *clusterv1.Clu
 		return serviceAnnotation, nil
 	}
 
-	// TODO(iXinqi): Temporarily commenting adding annotation to HA svc, will uncomment it after the feature is fully tested.
-	//aviInfraSetting, err := r.getAviInfraSettingFromCluster(ctx, cluster)
-	//if err != nil {
-	//	return serviceAnnotation, err
-	//}
-
-	//if aviInfraSetting != nil {
-	//	// add AVIInfraSetting annotation when creating HA svc
-	//	serviceAnnotation[akoov1alpha1.HAAVIInfraSettingAnnotationsKey] = aviInfraSetting.Name
-	//}
+	//check if mgmt cluster and then if there should a ais created from adc, then wait
+	if _, ok := cluster.Labels[akoov1alpha1.ManagememtClusterRoleLabel]; ok {
```

Comment:
What happens to workload clusters? Shouldn't we also annotate workload clusters' HA load balancer type of service?